index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
```

Comment:
I would suggest that loading two separate icon libraries for a total of three icons is not a great choice for the performance of your application. (Performance is a catch-all term we use to discuss how quickly things display in a web browser, how many files/assets a user has to download to interact with things.)

This means the user has to download four CSS files and, in the case of the webfonts, those CSS files request additional font files. You can see what this looks like in the Network tab of your dev tools (this screenshot is from Firefox) :
<img width="1439" alt="Screen Shot 2020-11-17 at 9 07 59 PM" src="https://user-images.githubusercontent.com/2653980/99473934-9ffcd400-2919-11eb-90a0-0ab87276edf0.png">

None of these files is individually very large, but these choices add up across a project. Ideally, you would either:
- use _only_ Material Icons or only Font Awesome
- use SVGs for your icons